package.py

```diff
@@ -14,6 +15,31 @@
 ADDON = "seeq_plot_curve"
 custom_env = {"PATH": os.environ.get("PATH")}
 
+def configure_version(): 
+    data = None
+    with open ('addon.json', 'r') as file: 
+        data = json.load(file)
+    
+    current_version = data['version']
+    new_version = __version__
+    print("Existing addon version: ", current_version, " incoming addon version: ", new_version)
+    final_version = current_version
+    # This step will capture version changes in seeq.addons.plot_curve.__version__
+    # but the addon.json file still will need manual update as good practice for version control 
+    if current_version < new_version: 
```

Comment:
These lexical string comparisons won't work reliably. For example...
```
>>> print("1.10.0" < "1.2.0") 
True
```

Reply:
I've updated it with a `!=` instead since if they're different for any reason we'll want to default to the value of `__version__`